src/applications/vaos/tests/e2e/va-appointment-eye.cypress.spec.js

```diff
@@ -0,0 +1,54 @@
+import {
+  initAppointmentListMock,
+  initVAAppointmentMock,
+} from './vaos-cypress-helpers';
+import * as newApptTests from './vaos-cypress-schedule-appointment-helpers';
+
+describe('Appointment Request', () => {
+  beforeEach(() => {
+    initAppointmentListMock();
+    initVAAppointmentMock();
+    cy.visit('health-care/schedule-view-va-appointments/appointments/');
+    cy.get('.va-modal-body button').click();
+  });
+
+  it('should submit an a va appointment', () => {
+    // Start flow
+    cy.injectAxe();
```

Comment:
I think we need to use this function once, after `visit()` and then `cy.axeCheck()` on each page change.